server/bin/pbench-state-manager.py

```diff
@@ -1,302 +0,0 @@
-#!/usr/bin/env python3
-# -*- mode: python -*-
-
-"""Command line interface to Dataset state.
```

Comment:
Are we sure we're done with this?  (@dbutenhof??)

Reply:
There are no references to it anywhere that I can find.

Reply:
That's because it's for use by _humans_.  🤣